src/components/organisms/UiAccordion/_internal/UiAccordionItem.vue

```diff
@@ -105,8 +106,8 @@ const props = defineProps({
 });
 const defaultProps = computed(() => ({
   settings: {
-    iconOpen: 'chevron-up',
-    iconClose: 'chevron-down',
+    iconOpen: 'chevron-up' as IconAsString,
```

Comment:
Q: Icon can by [Object, String] here we support just IconAsString. Should we change this type?